Basic-Car-Maintenance/Shared/Dashboard/Views/VehicleSelectionView.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  VehicleSelectionView.swift
+//  Basic-Car-Maintenance
+//
+//  https://github.com/mikaelacaron/Basic-Car-Maintenance
+//  See LICENSE for license information.
+//
+
+import SwiftUI
+
+struct VehicleSelectionView: View {
+    @Environment(\.dismiss) var dismiss
+    @Binding var selectedVehicle: Vehicle?
+    var vehicles: [Vehicle]
```

Comment:
I order these by scope public is closer to the top, and private is closer to the bottom

so these would go in this order:
```suggestion
    @Binding var selectedVehicle: Vehicle?
    var vehicles: [Vehicle]
    @Environment(\.dismiss) var dismiss
```

usually I put the binding first and then whatever variable, and Environment (cause it's not passed in next